src/operator/rnn-inl.h

```diff
@@ -436,387 +563,901 @@ class RNNOp : public Operator{
     if (param_.state_outputs) {
       hy_ptr = out_data[rnn_enum::kStateOut].dptr<DType>();
     }
-    DType* cx_ptr = NULL;
-    DType* cy_ptr = NULL;
+    DType * cx_ptr = NULL;
+    DType * cy_ptr = NULL;
```

Comment:
`DType* cx_ptr` looks better and makes the style more consistent.

Reply:
fixed